### PR TITLE
Dev/nigelhew

### PR DESCRIPTION
**Pull request by nhew1994**
Updated docstrings in workflows.py. 

- Question: What is the return type in def charge_density_difference?

Small fixes to eos_fit.py

- Removed some redundant code. I think plot_mv should be moved to magnetism.py

Small reformatting changes to data_extraction.py and aggregate_extraction.py

### CODE REVIEW

File: dfttk/eos_fit.py [hunk 1]
```diff
@@ -1260,7 +1260,7 @@ def fit_to_all_eos(df: pd.DataFrame) -> tuple[pd.DataFrame, pd.DataFrame]:
     return eos_df, eos_parameters_df
 
 
-# TODO: Not sure if a list of dataframe will actually work. the function simply concats them all together
+# TODO: Consider moving to magnetism.py. Not related to EOS fitting.
```
Comment (lukeamyers) [inline dfttk/eos_fit.py:1263]: agreed

File: dfttk/workflows.py [hunk 2]
```diff
@@ -367,26 +367,24 @@ def ev_curve_series(
             os.remove(file_path)
 
 
+# TODO: What is the return type?
```
Comment (lukeamyers) [inline dfttk/workflows.py]: Chgcar

**Review by lukeamyers — APPROVED** [review]
~~Go ahead and copy paste the plot_mv code to magnetism, or I can make this change later.~~

I think changing it to "Chgcar" works with sphinx/napolean, but I'm not sure.

**Comment by lukeamyers** [comment]
Actually, let me move plot_mv in the other pull request.

File: dfttk/workflows.py (hunk 2 above)
Comment (nhew1994) [reply dfttk/workflows.py]: I added that in the latest commit.

**Comment by nhew1994** [comment]
> Actually, let me move plot_mv in the other pull request.

Thanks.

**Pull request merged**